docs/content/manual/backup-and-restore/4907-vm-backup-restore-longhorn-1.5-plus.md

```diff
@@ -0,0 +1,68 @@
+---
+title: VM backup restore use Longhorn v1.5+ attach/detach mechanism
+---
+
+* Related issues: [#4907](https://github.com/harvester/harvester/issues/4907) [ENHANCEMENT] Take advantage of the attach/detach mechanism in Longhorn v1.5+
+
+## Category: 
+* Backup and Restore
+
+## Verification Steps
+Case 1: snapshot can work on a stopped VM.
+1. Create a VM.
+1. After the VM is ready, stop the VM.
+1. Check VM volumes are detached.
+1. Take a snapshot on the VM. The snapshot can be ready.
+
+Case 2: restore a snapshot from detached volumes can work.
+1. Follow Case 1.
+1. Make sure VM volumes are detached.
+1. Restore the snapshot to a new VM. The new VM can be ready.
+1. Restore the snapshot to replace the old VM. The old VM can be ready.
+
+Case 3: backup can work on a stopped VM
+1. Create a VM.
+1. After the VM is ready, stop the VM.
+1. Check VM volumes are detached.
+1. Take a backup on the VM. The backup can be ready.
+
+Case 4: race condition doesn't break VMBackup
+1. Create a VM.
+1. After the VM is ready, stop the VM.
+1. Check VM volumes are detached.
+1. Take multiple backup on the VM in a short time. All backup can be ready.
+
+Case 5: restore a backup from detached volumes can work:
+1. Follow Case 3.
+1. Make sure VM volumes are detached.
+1. Restore the backup to a new VM. The new VM can be ready.
+1. Restore the backup to replace the old VM. The old VM can be ready.
+
+## Expected Results
+* Case 1: snapshot can work on a stopped VM 
+   - VM volumes are detached after we stop VM.
+      ```
+          node1:~ # kubectl get volume -A
+          NAMESPACE         NAME                                       STATE      ROBUSTNESS   SCHEDULED   SIZE          NODE   AGE
+          longhorn-system   pvc-5a861225-920d-4059-b501-f02b2fd0ff27   detached   unknown                  10737418240          19m
+      ```
```

Comment:
Decrease an indent.

Current
```
    node1:~ # kubec
    NAMESPACE
    ...
```

Should be

```
node1:~ # kubec
NAMESPACE
```

Reply:
Thanks for the check.
Decrease the indent to align the format.